synchronizer/src/endpoints.rs

```diff
@@ -21,11 +21,12 @@ pub(crate) async fn handler_get_created_item(
 ) -> Result<impl warp::Reply, warp::Rejection> {
     let item = RawValue::from_hex(&item_str).map_err(|e| CustomError(e.to_string()))?;
     let created_items = node.created_items.read().unwrap();
+    let epoch = node.created_item_roots_pair.lock().unwrap().0;
```

Comment:
this looks dangerous, the created_items and epoch are under different mutexes, so they could be inconsistent!
I think I'll just move all state into a single RwLock in a future PR